Makefile

```diff
@@ -8,21 +8,22 @@ $(TEST_DIR)/%.json: $(TEST_DIR)/%.cairo
 	cairo-compile --cairo_path="$(TEST_DIR):$(BENCH_DIR)" $< --output $@
 
 deps:
+	CFLAGS=-I/opt/homebrew/opt/gmp/include LDFLAGS=-L/opt/homebrew/opt/gmp/lib pip install fastecdsa
+	pip install ecdsa fastecdsa sympy cairo-lang==0.9.1 maturin
 	python3 -m venv cairo-rs-py-env
 	pyenv install pypy3.7-7.3.9
 	export PYENV_VERSION=pypy3.7-7.3.9
-	pip install cairo_lang==0.9.1
 
 deps-macos:
+	CFLAGS=-I/opt/homebrew/opt/gmp/include LDFLAGS=-L/opt/homebrew/opt/gmp/lib pip install fastecdsa
+	pip install ecdsa fastecdsa sympy cairo-lang==0.9.1 maturin
 	python3 -m venv cairo-rs-py-env
 	pyenv install pypy3.7-7.3.9
 	export PYENV_VERSION=pypy3.7-7.3.9
-	CFLAGS=-I/opt/homebrew/opt/gmp/include LDFLAGS=-L/opt/homebrew/opt/gmp/lib pip install fastecdsa
-	pip install cairo_lang==0.9.1
```

Comment:
Why was this moved back into the test targets?

Reply:
We need to install `cairo_lang ` inside the virtual environment, otherwise when running cairo_rs_py, imports from starkware files are not recognized

Reply:
We need to install `cairo_lang` inside the virtual environment, otherwise cairo_rs_py is not able to recognize imports from starkware (aka importing stuff from starkware files inside hints)

Reply:
But doesn't that belong to the env creation?